src/main/resources/defaults/data/SkillParticleGeneration.json

```diff
@@ -515,7 +515,11 @@
 		"amountList": [
 			{
 				"value": 4,
-				"chance": 100
+				"chance": 70
+			},
+			{
+				"value": 3,
+				"chance": 30
```

Comment:
Do you have a source for this? I know these are the rates that some leaks mentioned, but `BinOutput/Ability/Temp/AvatarAbilities/ConfigAbility_Avatar_Yelan.json` does not support this. The data in there says that she always generates 12 energy, i.e. 4 hydro particles.

Reply:
Theres always a chance it changed from beta, however ubacha has claimed that theres a chance for it to generate 3-4 depending on hold and press skill. Heres the tweet. Unless his source lied it'd be weird if it generated anything else https://twitter.com/Ubatcha1/status/1509954923357777921?s=20&t=x5TRifdlh_japR82Ju4Oyg

Reply:
Yes, that is the leak I mentioned. There was also an earlier one about her always generating four, from an earlier version of the beta. It is possible that they just went back to that for the release.

For now, I would suggest sticking to what the data in `BinOutput` says. If tests on the live server show different behavior, we can always change it then.

Reply:
👍 got it